flink/v1.18/flink/src/test/java/org/apache/iceberg/flink/TestRowDataWrapper.java

```diff
@@ -49,12 +49,12 @@ public void testTime() {
               return;
             }
 
-            Assertions.assertThat(actual).isNotNull();
-            Assertions.assertThat(expected).isNotNull();
+            assertThat(actual).isNotNull();
+            assertThat(expected).isNotNull();
 
             int expectedMilliseconds = (int) ((long) expected / 1000_000);
             int actualMilliseconds = (int) ((long) actual / 1000_000);
-            Assert.assertEquals(message, expectedMilliseconds, actualMilliseconds);
+            assertThat(expectedMilliseconds).as(message).isEqualTo(actualMilliseconds);
```

Comment:
the actual/expected is the wrong way around. It should be `assertThat(actualMilliseconds).isEqualTo(expectedMilliseconds)`